src/modules/likes.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+class Likes {
+  constructor(item_id) {
+    this.item_id = item_id;
+  }
+}
+
+export default Likes;
```

Comment:
- [ ] Let's make sure likes are displayed when the pages is loaded. Here when page is loaded there is zero like but when I like the meal once likes became 17.
Before like:

![Screenshot from 2022-10-29 15-11-50](https://user-images.githubusercontent.com/65860678/198833573-7ffaa1fe-4819-45fb-847e-28a500d9021c.png)
After like:

![Screenshot from 2022-10-29 15-12-03](https://user-images.githubusercontent.com/65860678/198833577-dd304990-fb30-42e7-9773-d79359c8b395.png)

Reply:
Thanks for pointing this out, I have been able to implement this now